test/user.test.js

```diff
@@ -0,0 +1,57 @@
+const request = require('supertest');
+
+const app = require('../src/backend/web/app');
+
+jest.mock('../src/backend/utils/elastic');
+
+// Mock the internal authentication strategy
+jest.mock('../src/backend/web/authentication');
+// Use our authentication test helper
+const { login, logout } = require('./lib/authentication');
+
+describe.skip('test GET /user/feeds endpoint', () => {
+  let user;
+
+  beforeAll(() => {
+    user = login('Johannes Kepler', 'user1@example.com');
+  });
+
+  beforeEach(() => {
+    logout();
+    login('Johannes Kepler', 'user1@example.com');
+  });
+
+  afterAll(() => logout());
+
+  it('should respond with a 403 status when not logged in', async () => {
+    logout();
+    const res = await request(app).get(`/user/feeds`);
+    expect(res.status).toEqual(403);
+  });
+
+  it('should respond with a 200 status and JSON array if logged in', async () => {
+    const res = await request(app).get(`/user/feeds`);
+    expect(res.status).toEqual(200);
+    expect(res.get('Content-type')).toContain('application/json');
+    expect(res.body instanceof Array).toBe(true);
+  });
+
+  it('should respond with an updated array after a new user feed is added/removed', async () => {
+    const feedCount = (await request(app).get(`/user/feeds`)).body.length;
```

Comment:
Let's login here again before we start.

Reply:
Fixed in https://github.com/Seneca-CDOT/telescope/pull/958/commits/5cee6b4c37a89f88fa372ea01b50ff1e5cf45b72